.github/workflows/ci.yml

```
@@ -93,3 +93,13 @@ jobs:
               with:
                   extra_args: --only-verified
               continue-on-error: true
+
+            - name: Socket.dev supply chain scan
+              if: false
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)